biapol_utilities/label/_label_overlap_matrix.py

```diff
@@ -17,21 +19,17 @@ def label_overlap_matrix(label_image_x, label_image_y):
     Returns
     -------
     overlap: ND-array, int
-        matrix of pixel overlaps of size [x.max()+1, y.max()+1]
+        matrix of pixel overlaps of size [max(x, y) + 1, max(x, y) + 1], with 
+        entry result[i, j] corresponding to the overlap of label i in image x 
+        with label j in image y
     """
-    # put label arrays into standard form then flatten them 
-#     x = (utils.format_labels(x)).ravel()
-#     y = (utils.format_labels(y)).ravel()
+    
+    
+    # Flatten input arrays
```

Comment:
```suggestion
    # Make input arrays 1D
```
... in ImageJ "Flatten" means something very different. That's why I'm suggesting to be more precise here.